docs/coding-guidelines/package-projects.md

```diff
@@ -214,9 +214,10 @@ Sample `System.IO.FileSystem.pkgproj`
 ```
 
 ## Asset selection
+
 The makeup of a package folder is primarily a grouping of project references to the projects that compose that package.  Settings within each referenced project determines where that asset will be placed in the package.  For example, reference assembly projects will be placed under the `ref/{targetMoniker}` folder in the package and implementations will be under either `lib/{targetMoniker}` or `runtimes/{rid}/lib/{targetMoniker}`.  Whenever NuGet evaluates a package in the context of a referencing project it will choose the best compile time asset (preferring `ref`, then falling back to `lib`) and runtime asset (preferring `runtimes/{rid}/lib` and falling back to `lib`) for every package that is referenced.  For more information see http://docs.nuget.org/.
```

Comment:
Side-note: A lot of outdated info in this file. E.g. the examples should be SDK projects.

Reply:
Yes, I think every file (at least every one we expect to be up to date and matter) should be rated for freshness, and work parceled out.  That would be a good thing to do in #304